auth.json

```diff
@@ -0,0 +1,12 @@
+{
+  "http-basic": {
+    "repo.magento.com": {
+      "username": "8d8b26b69cc64ebaa3687fde19f69a51",
+      "password": "7970da75e8347fb327100f9066735f7d"
+    },
+    "packages.trollweb.no": {
+      "username": "trollweb",
+      "password": "OcIIPtJRtJ714jb2hWxHHjtu27RAdlarjqLULv7"
```

Comment:
Committing these secrets seems like spectacularly bad idea?

Reply:
This is absolutely my fault. Somehow I overlooked it. It's great that you noticed it.